cryptomator/src/main/java/ch/cyberduck/core/cryptomator/CryptoVault.java

```diff
@@ -363,6 +368,27 @@ protected void open(final VaultConfig vaultConfig, final CharSequence passphrase
         }
     }
 
+    protected void open(final VaultConfig vaultConfig, final Masterkey masterKey) throws BackgroundException {
+        this.open(vaultConfig, masterKey, createFilenameProvider(vaultConfig), createDirectoryProvider(vaultConfig));
```

Comment:
Although redundant we have the code style in this code base to always prefix with `this.` for instance methods.

Reply:
done